### PR TITLE
Support openWrt ARM images

### PR DESCRIPTION
**Pull request by monstermunchkin**

### CODE REVIEW

**Comment by monstermunchkin** [comment]
This closes #269

File: jenkins/jobs/image-openwrt.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@
         type: slave
         values:
         - amd64
+        - aarch64
+        - armv7l
```
Comment (stgraber) [inline jenkins/jobs/image-openwrt.yaml:15]: Those aren't valid architectures, we'd expect `arm64` and `armhf` but that will cause the image being built on the native architecture which from what you've mentioned over on the distrobuilder issue won't actually work.

Comment (monstermunchkin) [reply jenkins/jobs/image-openwrt.yaml:15]: Should we then close this PR, and just have users build the images themselves with `distrobuilder`?

Comment (stgraber) [reply jenkins/jobs/image-openwrt.yaml:15]: We can merge the yaml change and just skip the jenkins change so they at least have a YAML they can use.

Comment (monstermunchkin) [reply jenkins/jobs/image-openwrt.yaml]: Done.

**Pull request merged**